code/modules/tgui/tgui.dm

```diff
@@ -203,9 +203,8 @@
 		return
 	if(!COOLDOWN_FINISHED(src, refresh_cooldown))
 		refreshing = TRUE
-		addtimer(CALLBACK(src, .proc/send_full_update), TGUI_REFRESH_FULL_UPDATE_COOLDOWN, TIMER_UNIQUE)
+		addtimer(CALLBACK(src, .proc/send_full_update), COOLDOWN_TIMELEFT(src, refresh_cooldown), TIMER_UNIQUE)
```

Comment:
If full update is called every second, why bother having a static data instead of just putting everything on regular ``ui_data``?

Reply:
This timer isn't to invoke `send_full_update` every second--it's called specifically only when `refresh_cooldown` isn't done (e.g., cooldown not finished, check back when it is)

Reply:
To clarify: static data shouldn't be getting sent every second. I don't think it is--unless I'm wrong?

Reply:
No you're fine, I just misread and thought it would constantly be calling this.